google/cloud/spanner/value.h

```diff
@@ -523,7 +523,7 @@ class Value {
     google::spanner::v1::Type const& type;
     template <typename T>
     void operator()(T& t) {
-      auto value = GetValue(T{}, list_value.values(i), type);
+      auto value = GetValue(T{}, list_value.values(static_cast<int>(i)), type);
```

Comment:
I think the declaration of `ExtractTupleValues::i` on line 521 should probably declare that member as type `int` rather than `std::size_t`. size_t *seems* like the right type, but since this is a protobuf, the actual right type is `int`.

So I think if you change that declaration, you won't need these casts, and the code should work fine and indeed, probably be slightly more correct (since these are protos).

Reply:
Thanks! Done